src/nbsap/settings.py.docker

```diff
@@ -0,0 +1,256 @@
+import sys
+import os
+
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+DEBUG = os.environ.get('DEBUG', False) and True
+TEMPLATE_DEBUG = DEBUG
+
+ADMINS = (
+    # ('Your Name', 'your_email@example.com'),
+)
+
+MANAGERS = ADMINS
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.mysql', # Add 'postgresql_psycopg2', 'mysql', 'sqlite3' or 'oracle'.
+        'NAME': 'nbsap',                      # Or path to database file if using sqlite3.
+        # The following settings are not used with sqlite3:
+        'USER': 'nbsap',
+        'PASSWORD': 'nbsap',
```

Comment:
For the production setup, we will want to take the database configuration from the environment.